app/models/app_config.rb

```diff
@@ -10,7 +10,11 @@ class AppConfig < ActiveRecord::Base
   validates :site_title,   presence: true,
                            length: { maximum: 20 }
   validates :admin_email,
+            presence: true,
             format: { with: /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i }
+  validates :contact_link_location,
+            allow_blank: true,
+            format: { with: /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/ }
```

Comment:
Should there be an i after the final backslash(as in the admin email) for case insensitive?